src/angular/components/register.component.ts

```diff
@@ -12,56 +12,26 @@ import { PasswordGenerationService } from '../../abstractions/passwordGeneration
 import { PlatformUtilsService } from '../../abstractions/platformUtils.service';
 import { StateService } from '../../abstractions/state.service';
 
+import {
+    ResetMasterPasswordComponent as BaseResetMasterPasswordComponent,
+} from '../../angular/components/reset-master-password.component';
```

Comment:
just use `} from './reset-master-password.component';` since these are in the same directory.